kornia/nerf/data_utils.py

```diff
@@ -121,10 +122,14 @@ def __getitem__(self, idxs: Union[int, List[int]]) -> RayGroup:
             A ray parameter object that includes ray origins, directions, and rgb values at the ray 2d pixel
             coordinates: RayGroup
         """
+        if not isinstance(self._ray_sampler, RaySampler):
```

Comment:
we have a `KORNIA_CHECK_TYPE` -- context is that we are moving towards using the KORNIA_CHECK* as much as possibly so that we can log properly all the errors. I'll work soon on a refactor to that api

Reply:
mypy can't infer that after `KORNIA_CHECK_TYPE` the type will be correct